src/stores/widgets/StopGapWidgetDriver.ts

```diff
@@ -169,7 +170,7 @@ export class StopGapWidgetDriver extends WidgetDriver {
                     if (containsEmoji(content, effect.emojis)) {
                         // For initial threads launch, chat effects are disabled
                         // see #19731
-                        const isNotThread = content["m.relates_to"].rel_type !== RelationType.Thread;
+                        const isNotThread = content["m.relates_to"].rel_type !== THREAD_RELATION_TYPE.name;
```

Comment:
Shouldn't this use `THREAD_RELATION_TYPE.matches(relation?.rel_type)` to support both legacy and new?